src/pages/public_registration/ChapterPublisherInfo.tsx

```diff
@@ -0,0 +1,52 @@
+import { Box, Typography } from '@mui/material';
+import { useQuery } from '@tanstack/react-query';
+import { useTranslation } from 'react-i18next';
+import { fetchResults } from '../../api/searchApi';
+import { BookPublicationContext } from '../../types/publication_types/bookRegistration.types';
+import { ChapterPublicationContext } from '../../types/publication_types/chapterRegistration.types';
+import { ReportPublicationContext } from '../../types/publication_types/reportRegistration.types';
+import { getIdentifierFromId } from '../../utils/general-helpers';
+import { PublicPublisher } from './PublicPublicationContext';
+
+interface ChapterPublisherInfoProps {
+  publicationContext: ChapterPublicationContext;
+}
+
+export const ChapterPublisherInfo = ({ publicationContext }: ChapterPublisherInfoProps) => {
+  const { t } = useTranslation();
+
+  const publisherIdentifier = publicationContext.id && getIdentifierFromId(publicationContext.id);
+
+  const publisherQuery = useQuery({
+    enabled: !!publisherIdentifier,
+    queryKey: ['publisher', publisherIdentifier],
+    queryFn: () => fetchResults({ id: publisherIdentifier, results: 1 }),
+    meta: { errorMessage: t('feedback.error.search') },
+  });
+
+  const publisherQueryResult = publisherQuery.data?.hits[0];
+
+  const bookPublicationContext =
+    publisherQueryResult &&
+    (publisherQueryResult?.entityDescription?.reference?.publicationContext as
+      | BookPublicationContext
+      | ReportPublicationContext);
+
+  return (
+    <>
+      {bookPublicationContext && (
+        <>
+          {bookPublicationContext?.isbnList && (
```

Comment:
allerede sjekket at denne ikke er tom over. kanskje btw kanskje renames til `publisherPublicationContext` elns også, siden det nå er både bok og rapport
```suggestion
          {bookPublicationContext.isbnList && (
```